docs/index.md

```diff
@@ -51,9 +51,10 @@ This list showcases upcoming UK student-run hackathons and hackathon-associated
 | CreatED Hack '20 | [createdhack.com](https://createdhack.com/) | University of Edinburgh | 200 | 14th-15th March 2020 |
 | HackMed   | [hackmed.uk](http://hackmed.uk/) | University of Sheffield | ?? |  14th-15th March 2020 |
 | HackThePlug   | [hacktheplug.tech](https://hacktheplug.tech/) | University of Bath | ?? |  28th-29th March 2020 |
+| Hackathons UK Unconference | [hackathonhackers.eu](https://hackathonhackers.eu) |  Manchester Metropolitan University  | ~50 | 4th April 2020 |
```

Comment:
I like this, but... can we have no URL (e.g. `TBC`) until we have a proper website up?